test/suite_test.go

```diff
@@ -105,11 +107,21 @@ func (e *kinde2e) SetupSuite() {
 	e.Nil(os.MkdirAll(buildDir, 0o755))
 
 	e.kindPath = filepath.Join(buildDir, "kind")
-	e.downloadAndVerify(
-		"https://github.com/kubernetes-sigs/kind/releases/download/"+
-			kindVersion+"/kind-linux-amd64",
-		e.kindPath, kindSHA512,
-	)
+	operatingSystem := runtime.GOOS
+	switch operatingSystem {
```

Comment:
No need to bind a variable here 
```suggestion
	switch runtime.GOOS {
```